doc/classes/TextEdit.xml

```diff
@@ -83,15 +83,14 @@
 				Adds a selection and a caret for the next occurrence of the current selection. If there is no active selection, selects word under caret.
 			</description>
 		</method>
-		<method name="adjust_carets_after_edit">
+		<method name="adjust_carets_after_edit" deprecated="No longer necessary since methods now adjust carets themselves.">
 			<return type="void" />
 			<param index="0" name="caret" type="int" />
 			<param index="1" name="from_line" type="int" />
 			<param index="2" name="from_col" type="int" />
 			<param index="3" name="to_line" type="int" />
 			<param index="4" name="to_col" type="int" />
 			<description>
-				Reposition the carets affected by the edit. This assumes edits are applied in edit order, see [method get_caret_index_edit_order].
 			</description>
```

Comment:
`adjust_carets_after_edit` no longer does anything, and shouldn't do anything, since its behavior is essentially part of the other edit methods, like `remove_text()`.

Reply:
If it does nothing the description needs to be updated to explicitly state it ("This method does nothing.") for the same reasons as above.

Reply:
I can also restore the descriptions I removed of get_selection_line/column.